app/locale/fr.coffee

```diff
@@ -276,9 +276,9 @@ module.exports = nativeDescription: "français", englishDescription: "French", t
     view_progress: "Voir Progrès"
     go_to_courses: "Aller à Mes Cours"
     want_coco: "Voulez-vous CodeCombat à votre école?"
-#    educator: "Educator"
-#    student: "Student"
-#    go_to_my_dashboard: "Go to My Dashboard"
+   educator: "Educateur"
```

Comment:
there is a spacing issue with translations in this pull request. You need to add extra space to make `want_coco` and `educator` at same indentation level to make this a valid file.

You need to fix the indents for all the transitions in this PR